test-support/reference-trie/Cargo.toml

```diff
@@ -1,22 +1,22 @@
 [package]
 name = "reference-trie"
-version = "0.13.0"
```

Comment:
this is technically a breaking change for test crates, is it ok?

Reply:
I wonder, the 0.2 from util-mem remove some functionality that were not used in trie crates, so the update is mainly to remove a dep related to this functionality from the trie crates, so if another project uses this functionality it will just depend on two version of util-mem.

In our case it seems quite unlikely to be an issue (cargo update still needed I think).

Reply:
Although the crates don't use the removed functionality, they publicly expose types from semver incompatible versions, e.g.
```rust
pub type RefTrieDB<'a> = trie_db::TrieDB<'a, keccak_hasher::KeccakHasher, ReferenceNodeCodec>;
```

Reply:
Oh yes, let's reset it to 0.14 (I do not know why I tend to want to avoid increasing version number like that is costly).